lib/qiita/markdown/filters/greenmat.rb

```diff
@@ -16,13 +20,18 @@ def greenmat
             Qiita::Markdown::Greenmat::HTMLRenderer.new(hard_wrap: true, with_toc_data: true),
             autolink: true,
             fenced_code_blocks: true,
-            footnotes: true,
+            footnotes: options[:footnotes],
             no_intra_emphasis: true,
             no_mention_emphasis: true,
             strikethrough: true,
             tables: true,
           )
         end
+
+
+        def options
+          @options ||= DEFAULT_OPTIONS.merge(context[:greenmat] || {})
+        end
```

Comment:
From my understanding, the users of `SummaryProcessor` need to pass `{ greenmat: { footnotes: false } }` context explicitly with your implementation.

Rather than handling the default options in this `Greenmat` filter, how about handling it in `Processor` and `SummaryProcessor` [like this](https://github.com/increments/qiita-markdown/blob/v0.11.5/lib/qiita/markdown/processor.rb#L4-L23)? We want the default behavior (i.e. passing no context to `Qiita::Markdown::Processor/SummaryProcessor.new`) to match Qiita's Markdown spec.